tests/integration/persist_constraints/test_persist_constraints.py

```diff
@@ -89,27 +91,25 @@ def test_incremental_constraints(self):
         self.run_dbt(["run", "--select", model_name, "--full-refresh"])
         self.check_constraints(model_name, {"delta.constraints.id_greater_than_zero": "id > 0"})
 
-        schema = self.unique_schema()
-
         # Insert a row into the seed model with an invalid id.
         self.run_sql_file("insert_invalid_id.sql")
         self.run_and_check_failure(
             model_name,
             err_msg="CHECK constraint id_greater_than_zero",
         )
         self.check_staging_table_cleaned()
-        self.run_sql(f"delete from {schema}.seed where id = 0")
+        self.run_sql("delete from {database_schema}.seed where id = 0")
```

Comment:
Interesting, so if we use `{database_schema}` in the query then dbt will automatically change it to use the target database and target schema?

Reply:
Actually I made some changes in `tests/integration/base.py` to handle `database_schema`.

Reply:
:+1: it's in `transform_sql`